tests/test_crs/valid_fiod.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: benchmark.example.com/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  cleanup: false
+  workload:
+    name: "fio_distributed"
+    args:
+      # To disable fio, set clients to 0
+      servers: 1
```

Comment:
best to run with servers set 2 ?

Reply:
Why?

Reply:
We'd want to test with at least 2 servers so that we've better coverage say at a task like this `https://github.com/cloud-bulldozer/ripsaw/pull/74/files#diff-7aa81d437a25b88f200c4b77a2f138f9R32`